collector/lib/ProcessSignalHandler.cpp

```diff
@@ -33,6 +35,11 @@ bool ProcessSignalHandler::Stop() {
 
 SignalHandler::Result ProcessSignalHandler::HandleSignal(sinsp_evt* evt) {
   const auto* signal_msg = formatter_.ToProtoMessage(evt);
+  const char* name = signal_msg->signal().process_signal().name().c_str();
```

Comment:
now that I look again, I think we want to put this after line 43 (after we've checked the `signal_msg` pointer)